grails-app/views/email/_approvalsAlert.gsp

```diff
@@ -11,11 +12,32 @@
     <div>
         <g:message
                 code="email.requestReceived.message"
-                args="[requisition.destination,
-                       requisition.requestedBy,
-                       g.createLink(uri: redirectUrl, absolute: true),
+                args="[requisition.destination]"
+        />
+        %{--
+             If there is a desired delivery date we want to add a comma at the end of this message, because there is a
+             second part of the sentence. If there is no delivery date, we just finish the sentence after requestCreatedBy
+             message. It cannot be included in HTML alone because there will be additional space around the punctuation mark.
+         --}%
+        <g:set var="punctuationMark" value="${requisition.desiredDeliveryDate ? ',' : '.'}" />
```

Comment:
I get the idea here, but this looks odd. Would it not be clearer if it was just a `{requisition.desiredDeliveryDate ? ',' : '.'}` in between two `g:message` tags instead of an additional argument?

Reply:
or the other option would be always having the 
```You have received a new request from: {0}, created by: {1}, desired date of delivery: {2}. Click <a href="{2}">here</a> to review your requests pending your approval.```
and `{2}` would get `requisition.desiredDeliveryDate ?: 'unknown'`, or something like that.

Reply:
To your second comment: I asked Manon about that before and she answered it would be better to not show the date if it is null.
To your first comment: As you can see I left a comment "It cannot be included in HTML alone because there will be additional space around the punctuation mark."

Reply:
ok